### PR TITLE
Response video search error

### LINKED ISSUE #1007: Response Video Search Error
**Description**
There's a 500 error when searching on response video view. 

**How to reproduce**
Steps to reproduce the behavior.
1. Head to https://lookit.mit.edu/exp/studies/<study_id>/responses/attachments/
2. Use "Search" field with any string. 
3. Page will on show "Server Error (500)"

**Expected behavior**
Table will return result where the name of the video contains the search string. 

**Screenshots**
search

** Environment (OS, browser, branch, etc.):**
All/Any

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by okaycj**
Fixes 500 error found when searching in response video view.

Closes #1007

### CODE REVIEW

**Comment by becky-gilbert** [comment]
Fix looks great. And thanks for finding all of the other unquoted `sort`s and setting a default 👍 
I just have the one question about the `state` value.

**Comment by okaycj** [comment]
@becky-gilbert 

> I just have the one question about the `state` value.

I went back and took a look at any html where the value attribute on the input element wasn't wrapped in a quotes.  I found two more places could be "fixed".  I pushed the commit, but these ui changes always have unknown repercussions.

**Review by mekline — APPROVED** [review]
Approved assuming you are satisfied with any QA around the potential mysterious UI effects you mentioned :)

**Comment by becky-gilbert** [comment]
Thanks @okaycj Happy to help with UI testing for each of the changed values.

**Review by becky-gilbert — APPROVED** [review]

File: studies/templates/studies/study_list.html [hunk 1]
```diff
@@ -38,7 +38,7 @@ <h1>Manage Studies</h1>
                        placeholder="Filter name or description"
                        size="50"
                        type="text"/>
-                <input type="hidden" name="sort" value={{ sort }}/>
+                <input type="hidden" name="sort" value="{{ sort }}"/>
                 <input type="hidden" name="state" value={{ state }}/>
```
[the remark this answers is not in the document]
Comment (okaycj) [reply studies/templates/studies/study_list.html]: I think you might have missed this change in commit 612b846.

**Pull request merged**